.github/workflows/test.yml

```diff
@@ -97,3 +97,14 @@ jobs:
       uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
     - name: Run Test
       run: go test -v ./...
+
+  # The module isn't useful without cgo, but it still needs to successfully build. Broad cgo-less
```

Comment:
I'll confess, none of this made sense to me :-).  I think it might be a passive vs active voice thing?

Reply:
The only passive voice I see (in the way I read it) is "how it's implemented"--I do think it's rushed and scattered, though, and it shouldn't be hard to fix. 😄